K8SPXC-1005 document storing backups on azure (#14)
percona/k8spxc-docs: docs/backups.md

                credentialsSecret: my-cluster-name-backup-s3
                region: us-west-2
                endpointUrl: https://URL-OF-THE-S3-COMPATIBLE-STORAGE
                ...
            ```

        === "Azure Blob storage"

            The `backupSource` key should contain `destination` key equal to the
            Azure Blob [container](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#containers)

>             with `https://` prefix, followed by the necessary Azure

Author: ```suggestion
            name, followed by the necessary Azure
```

>             with `https://` prefix, followed by the necessary Azure

Author: ```suggestion
            name, followed by the backup name
```

>             with `https://` prefix, followed by the necessary Azure

Author: ```suggestion
            and backup name, followed by the necessary Azure
```

>             with a special `azure://` prefix, followed by the necessary Azure

Author: ```suggestion
            with `https://` prefix, followed by the necessary Azure
```

>             configuration keys, same as in `deploy/cr.yaml` file:

Author: ```suggestion
            
```

            ```yaml
            ...
            backupSource:

>               destination: https://AZURE-CONTAINER-NAME/BACKUP-NAME

Reviewer: @nmarukovich please verify this

Reviewer 2: The whole way looks like this 
https://engk8soperators.blob.core.windows.net/operator-testing/cluster1-2022-11-21-17:51:07-full

operator-testing = AZURE-CONTAINER-NAME

Reviewer 3: This should not work with `https://` prefix

Reviewer: @pooknull is `https://` not necessary or just wrong?

Reviewer 2: ```
apiVersion: pxc.percona.com/v1
kind: PerconaXtraDBClusterRestore
metadata:
  name: restore1
spec:
  pxcCluster: cluster1
  backupSource:
    destination: operator-testing/cluster1-2022-12-06-19:39:19-full
    azure:
      credentialsSecret: azure-secret
      container: operator-testing
      endpointUrl: https://engk8soperators.blob.core.windows.net
```

Reviewer 3: @egegunes It's wrong. We shouldn't use any prefixes for that

>               destination: https://AZURE-CONTAINER-NAME/BACKUP-NAME

Author: ```suggestion
              destination: AZURE-CONTAINER-NAME/BACKUP-NAME
```

>               destination: azure://AZURE-CONTAINER-NAME/BACKUP-NAME

Author: ```suggestion
              destination: https://AZURE-CONTAINER-NAME/BACKUP-NAME
```
